packages/paste-core/components/alert/src/index.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {ValueOf} from '@twilio-paste/types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {MediaObject, MediaFigure, MediaBody} from '@twilio-paste/media-object';
+import {Button} from '@twilio-paste/button';
+import {CloseIcon} from '@twilio-paste/icons/esm/CloseIcon';
+import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {WarningIcon} from '@twilio-paste/icons/esm/WarningIcon';
+
+export const ROLES = {
+  error: 'alert',
+  neutral: 'status',
+  warning: 'alert',
+} as const;
+export const VARIANTS = {
+  error: 'error',
+  neutral: 'neutral',
+  warning: 'warning',
+} as const;
+export const BACKGROUND_COLORS = {
+  error: 'colorBackgroundErrorLightest',
+  neutral: 'colorBackgroundNeutralLightest',
+  warning: 'colorBackgroundWarningLightest',
+} as const;
+export const BORDER_COLORS = {
+  error: 'colorBorderErrorLight',
+  neutral: 'colorBorderNeutralLight',
+  warning: 'colorBorderWarningLight',
+} as const;
+
+export type Variants = ValueOf<typeof VARIANTS>;
+export type BackgroundColors = ValueOf<typeof BACKGROUND_COLORS>;
+export type BorderColors = ValueOf<typeof BORDER_COLORS>;
+export type Roles = ValueOf<typeof ROLES>;
+
+export interface AlertProps {
+  id?: never;
+  className?: never;
+  children: NonNullable<React.ReactNode>;
+  onDismiss?: () => void;
+  role?: string;
```

Comment:
Just curious, this can't be Roles because there are possibly other types of roles that would work here?

Reply:
Yup, it's basically a free for all is you really want to do still things other than making alerts and status'